code/script.js

```diff
@@ -1,47 +1,205 @@
 // All the DOM selectors stored as short variables
-const chat = document.getElementById('chat')
+const theWholeForm = document.getElementById('form-wrapper');
+const submitButton = document.getElementById('submit-ID') 
 
-// Global variables, if you need any, declared here
+//Global variables
+let questionNumber = 1; //stores the question number
+let TypeOfArrangement = ''; //stores type of arrangement to print in the order confirmation
 
-// Functions declared here
+//questionHandler function, passes the input from the user, 
+//plays sound effect and calls the functiond depending on the question number 
+const questionHandler = (userInput) => {
+  showMessage(userInput, 'user');
+  playSoundUser ()
+  switch (questionNumber) {
+    case 1: 
+      setTimeout(() => showFlowerArrangement(userInput), 1000); 
+      break;
+    case 2: 
+      setTimeout(() => showTypeOfArrangement(userInput), 1000);
+      break;
+    case 3: 
+      setTimeout(() => showFlowerSelector (userInput), 1500);
+      break;
+    case 4: 
+      setTimeout(() => showFinalConfirmation (userInput), 2000);
+      break;
+    default:
+      setTimeout(() => thankYou(), 1000);
+      break;
+  }
```

Comment:
I love that you used `switch` instead of `if/else`... It makes your code so clean 😻